dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlannerConfig.cs

```diff
@@ -53,10 +54,33 @@ public sealed class StepwisePlannerConfig
     /// <summary>
     /// The maximum number of iterations to allow in a plan.
     /// </summary>
-    public int MaxIterations { get; set; } = 100;
+    public int MaxIterations { get; set; } = 15;
 
     /// <summary>
     /// The minimum time to wait between iterations in milliseconds.
     /// </summary>
     public int MinIterationTimeMs { get; set; } = 0;
+
+    /// <summary>
+    /// Delegate to get the prompt template string.
+    /// </summary>
+    public Func<string>? GetPromptTemplate { get; set; } = null;
+
+    /// <summary>
+    /// The configuration to use for the prompt template.
+    /// </summary>
+    public PromptTemplateConfig? PromptUserConfig { get; set; } = null;
+
+    /// <summary>
+    /// A suffix to use within the default prompt template.
+    /// </summary>
+    public string Suffix { get; set; } = @"Let's break down the problem step by step and think about the best approach. Label steps as they are taken.
+
+Continue the thought process!";
+    #endregion Execution configuration
+
+    /// <summary>
+    /// A logger factory to use for logging.
+    /// </summary>
+    public ILoggerFactory? LoggerFactory { get; set; } = null;
```

Comment:
I added this before when it was a Logger. Probably makes sense to remove this and just use the kernel logger factory. Thoughts appreciated.

Reply:
Removing for now. Will add if folks think appropriate.